### PR TITLE
fixed typo keeping backward compatibility

### PR DESCRIPTION
**Pull request by abhinavdalal-iconnect**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -45,9 +45,10 @@ export interface CarouselProps {
   renderDotsOutside?: boolean; // show dots outside of the container for custom styling.
   renderButtonGroupOutside?: boolean; // show buttonGroup outside of the container for custom styling.
   // Show next/previous item partially
-  // partialVisbile has to be used in conjunction with the responsive props, details are in documentation.
+  // partialVisible has to be used in conjunction with the responsive props, details are in documentation.
   // it shows the next set of items partially, different from centerMode as it shows both.
-  partialVisbile?: boolean;
+  partialVisbile?: boolean; // old typo - deprecated
+  partialVisible?: boolean;
   customTransition?: string;
```
Comment (YIZHUANG) [inline src/types.ts]: I would like to remove this from the typing

File: stories/index.stories.js [hunk 2]
```diff
@@ -364,7 +364,7 @@ storiesOf("Carousel", module)
     return (
       <Carousel
         infinite
-        partialVisbile="right"
+        partialVisible="right" // should be boolean ideally
```
Comment (YIZHUANG) [inline stories/index.stories.js]: Yeah, this should be changed to boolean

**Comment by abhinavdalal-iconnect** [comment]
@YIZHUANG comments addressed

**Comment by abhinavdalal-iconnect** [comment]
I dont know about removing it from the code base completely.. that would make this unnecessarily major release since it would break backwards compatibility.. i think maybe we leave it in the typing and add it to tech debt somewhere; and then when ever we do a 3.0 release along with some genuine major changes.. we just remove the old typo? if you agree ill add it back to the typing with deprecated comment

**Comment by abhinavdalal-iconnect** [comment]
I have added a log if user is using the old typo prop.. so hopefully this will also give users a chance to change their props before they upgrade.. where should we maintain the tech debt? i usually just added it as a note in github project attached to repo.. if you can create a project then ill add the note..

**Comment by YIZHUANG** [comment]
@abhinavdalal-iconnect  Yeah, do that.

File: stories/index.stories.js (hunk 2 above)
Comment (ardian-c) [reply stories/index.stories.js]: @YIZHUANG How do you do the partial visibility to the left now?

Comment (abhinavdalal-iconnect) [reply stories/index.stories.js]: @ardian-c I think partialVisibility on the left can be achieved easily via centerMode (with infinite).. i dont think partial visibility on the left without centerMode(with infinite) makes sense; but if you absolutely need it you could try passing a negative additionalTransform ..

Comment (ardian-c) [reply stories/index.stories.js]: @abhinavdalal-iconnect thank you, will try it with negative `additionalTransform`, as I'm not sure if I understand correctly `centerMode` with infinite.

**Pull request merged**